common/softwarecontainer-common.h

```diff
@@ -40,7 +42,7 @@ static constexpr ContainerID INVALID_CONTAINER_ID = -1;
 static constexpr pid_t INVALID_PID = -1;
 static constexpr int INVALID_FD = -1;
 
-typedef std::map<std::string, std::string> GatewayConfiguration;
+typedef std::map<std::string, json_t *> GatewayConfiguration;
```

Comment:
I still think this should be a class so we can hide raw types from external dependencies more easily.